core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -190,13 +198,14 @@ protected void doStop() throws Exception {
 
     List<URI> listeners = parseListeners(config.getList(RestConfig.LISTENERS_CONFIG),
             config.getInt(RestConfig.PORT_CONFIG), Arrays.asList("http", "https"), "http");
+    SslContextFactory sslContextFactory = new SslContextFactory();
+    this.sslContextFactory = sslContextFactory;
```

Comment:
i think its cleaner to either
- do this in the constructor
- do this lazily in the getter

will prevent any races here.